components/rating-toast/toast.css

```diff
@@ -0,0 +1,60 @@
+:root{
+    --white:white;
+    --white-light:rgb(219, 208, 208);
+    --grey-dark:rgb(53, 49, 49);
+    --purple-light:rgb(155, 84, 221);
+
+}
+
+.main-toast {
+    width: 80%;
+    height: 100px;
```

Comment:
Use rem ?

Reply:
I have updated the unit, thnks for review